cmd.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"github.com/flynn-archive/go-shlex"
+	"github.com/pkg/errors"
+)
+
+// parseCmd is a wrapper function for the underlying shell lexer implementation,
+// it splits a commands string into argv slice based upon shell quoting and escaping
+// rules.
+func parseCmd(cmd string) ([]string, error) {
+	v, err := shlex.Split(cmd)
+	return v, errors.Wrap(err, "shlex: failed to parse command")
```

Comment:
@boumenot the way you pass a command directly to OS can be through `/bin/bash -c $CMD` and in that case you are giving shell access to the program. It is basically a security flaw. Also doing shlex is preserving the existing behavior.